### ISSUE TITLE
mark the VPN provider in the results

### ISSUE THREAD

**Issue by rpanah** [iclab/centinel#232]
Mark the name of the VPN service provider in the results. It would also help if we could include this information in the database table for the clients.

**Comment by rpanah** [comment]
@lianke123321, was this added? I remember seeing this in the results database...

**Comment by lianke123321** [comment]
Nope, previously I added a separate json file to map client to VPN provider as a temporary solution. Including VPN provider info in result is not done.
